tests/monotouch-test/UIKit/UIContentSizeCategoryTest.cs

```diff
@@ -45,6 +45,17 @@ public void Compare ()
 			Assert.Throws<ArgumentException> (() => UIContentSizeCategoryExtensions.Compare (small, (UIContentSizeCategory)271828));
 			Assert.Throws<ArgumentException> (() => ((UIContentSizeCategory)1234).IsAccessibilityCategory ());
 		}
+
+		[Test]
+		public void GetPreferredContentSizeCategoryTest ()
+		{
+			var sizeNSString = UIApplication.SharedApplication.PreferredContentSizeCategory;
+            var sizeEnum = UIContentSizeCategoryExtensions.GetValue(sizeNSString);
```

Comment:
```suggestion
            var sizeEnum = UIContentSizeCategoryExtensions.GetValue (sizeNSString);
```
small style change (nit)